pkgs/applications/networking/p2p/lokinet/default.nix

```diff
@@ -0,0 +1,62 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, cmake
+, libevent
+, libsodium
+, libuv
+, pkg-config
+, sqlite
+, systemd
+, unbound
+, zeromq
+}:
+
+stdenv.mkDerivation rec {
+  pname = "lokinet";
+  version = "0.9.9";
+
+  src = fetchFromGitHub {
+    owner = "oxen-io";
+    repo = "lokinet";
+    rev = "v${version}";
+    sha256 = "AaGsRg9S9Cng9emI/mN09QSOIRbE+x3916clWAwLnRs=";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [
+    cmake
+    pkg-config
+    systemd
```

Comment:
What is this user for?

Reply:
`systemd` integration allows for lokinet to setup DNS automatically, if you use `networkd`.

Reply:
Cool! So this belongs in `buildInputs` then.